src/setup/scss/_globals.scss

```diff
@@ -0,0 +1,21 @@
+/* stylelint-disable selector-class-pattern, selector-max-id */
+// This file is supposed to only hold styles, that are scoped by a globally used class.
+@use './mixins';
+@use './variables';
+
+#app {
+  min-height: 100%;
+}
+
+.invisible {
```

Comment:
Do we still have use for that? Only ever saw it used for hiding button labels and for that now we use usually `aria-label`.